lasagne/layers/base.py

```diff
@@ -200,7 +200,7 @@ def create_param(self, param, shape, name=None):
             if param.shape != shape:
                 raise RuntimeError("parameter array has shape %s, should be "
                                    "%s" % (param.shape, shape))
-            return theano.shared(param, name=name)
+            return theano.shared(utils.floatX(param), name=name)
```

Comment:
Without this it was creating float64s. It's less confusing if they're all casted using utils.floatX function. Obviously this section is changing anyways, but did this for now.